Source/WebKit/UIProcess/API/glib/WebKitNetworkSession.cpp

```diff
@@ -664,4 +664,66 @@ WebKitDownload* webkit_network_session_download_uri(WebKitNetworkSession* sessio
     return download.leakRef();
 }
 
+/**
+ * webkit_network_session_download_uri_finish:
+  * @session: a #WebKitNetworkSession
+  * @result: a #GAsyncResult
+  * @error: return location for error or %NULL to ignore
+  *
+  * Finish an asynchronous operation started with webkit_network_session_download_uri_async().
+  *
+  * Returns: (transfer full): a new #WebKitDownload representing the download operation.
+  * Since: 2.48
```

Comment:
```suggestion
  * Since: 2.50
```

Will need to change the Since tags everywhere.